examples/app/index.js

```diff
@@ -1,43 +0,0 @@
-// Copyright (c) Jupyter Development Team.
-// Distributed under the terms of the Modified BSD License.
-
-// ES6 Promise polyfill
-require('es6-promise/auto');
-
-// Load the core theming before any other package.
```

Comment:
The state of the example [here](https://github.com/jupyterlab/jupyterlab/tree/4b654f3bbc2c8a3ab5a19d862ca2006fe01e479c/examples/app) is enough to load an application that hard-codes a theme but does not handle settings.  We are going to move the settings handling to JupyterLab launcher, which is the intended way to handle custom spins.  cf https://github.com/jupyterlab/jupyterlab_launcher/issues/15